### PR TITLE
Group Index Left (Wearable): add example and style

### PR DESCRIPTION
**Pull request by grzegorz-czajkowski**
[Issue] https://github.com/Samsung/TAU/issues/684
[Problem] No support for group index left
[Solution] Add implementation and example (content only)

Signed-off-by: Grzegorz Czajkowski <g.czajkowski@samsung.com>

### CODE REVIEW

**Comment by grzegorz-czajkowski** [comment]
Implementation for content only. Needs to do the same thing for the list.

**Review by DohyungLim — COMMENTED** [review]
In my opinion, this is the code without any problems.

**Comment by grzegorz-czajkowski** [comment]
> In my opinion, this is the code without any problems.

Thanks. Soon I will extend this PR to support left adjustment for list as well.

File: examples/wearable/UIComponents/contents/presentation-views/list/group-index/multi-left.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+			</h2>
+		</header>
+		<div class="ui-content">
+			<ul class="ui-listview">
+				<li>
+					<a>
+						List 12
+					</a>
+				</li>
+				<li class="ui-li-group-index-left">
```
Comment (TomaszLukawskiSam) [inline examples/wearable/UIComponents/contents/presentation-views/list/group-index/multi-left.html]: I think you can try to do this in another way.
`<div class="ui-li-group-index ui-li-group-index-left">`

Comment (grzegorz-czajkowski) [reply examples/wearable/UIComponents/contents/presentation-views/list/group-index/multi-left.html]: Right. Changed examples and simplified styles. Thanks.

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks and works very well.

**Pull request merged**